### DISCUSSION TITLE
Help after fixing up ext4.vhdx

### DISCUSSION THREAD

**Discussion by SuperNestendo** [immich-app/immich#9251] [General]
Hey all,

I did a silly thing and now my Immich server is stuffed cause of it. My hard drive was running out of space so I followed [this reddit post](https://www.reddit.com/r/docker/comments/vx1vx5/how_do_i_delete_or_get_rid_of_ext4vhdx/) to clear out the monster that was talking half of my drive.

I _thought_ I had safely backed up the Immich server but alas, I have not. Now I've started it back up and all my data is missing. It is still present on my hard drive in the library folder so at least it's not disappeared. I've been looking on how to link this library back up to my server and the only solution I have found is using the CLI to re-upload everything. Is there a better solution or am I stuck doing this method?

**Comment by mmomjian** [comment]
If you lost your database, with no backup, the only option is to reupload everything.
